Added Arbitrary values for pitch bend up and down (1-48 semitone range)

twisted-electrons opened the pull request:
Added dropdown menus for pitch bend up and down ranges to the tool
https://twisted-electrons.com/tool/therapSID/

The values are being transmitted to and from the tool OK but I'm strugging with the actual implmentation of the bend.
The ranges are off (they seem to bend too far down and not far enough up)...

thomasj commented:
Found some ways to fix it properly, submitted #71 for this. Also changed the Tool commands to use same CC as sent (so web app needs updating to work again).

twisted-electrons commented:
Hi Thomas
Hope all is well. I fixed the tool and updated the new version.
Added a very small tweak:
TherapSID introduces itself to the tool with device ID#2 and FW version
digits before sending settings.
It's working wonders -Thanks again for your help,
Cheers
Alex




On Wed, Apr 26, 2023 at 11:15 PM Thomas Jansson ***@***.***>
wrote:

> Found some ways to fix it properly, submitted #71
> <https://github.com/twistedelectrons/TherapSID/pull/71> for this. Also
> changed the Tool commands to use same CC as sent (so web app needs updating
> to work again).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/twistedelectrons/TherapSID/pull/70#issuecomment-1524044308>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3MWKRUU3V2ZLRBF3VF7CJ3XDGEXBANCNFSM6AAAAAAXL6NZGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

thomasj commented:
Great!
